### PR TITLE
Add ImmutablesStyle annotation to remove warnings

### LINKED ISSUE #28: Immutable Style classes not found
The `auth-tokens` module uses `@Immutables` but does not include the annotations as compile time dependencies (it cannot find the styles classes). This causes downstream java warnings.

Two possible solutions would be to:

1. include the annotations in the classpath:

    ```
    compile group: 'org.immutables', name: 'value', classifier: 'annotations'
    ```

2. create and switch to an annotation provided in the `auth-tokens` module itself (i.e. something named `@ImmutableStyles`)

would a PR that does 2 make sense?

### PR DESCRIPTION
**Pull request by mnazbro**
Fixes #28

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,6 +4,9 @@ buildscript {
         maven {
             url 'https://plugins.gradle.org/m2/'
         }
+        maven {
+            url  "http://palantir.bintray.com/releases"
```
Comment (markelliot) [inline build.gradle]: why is this required?

**Comment by markelliot** [comment]
isn't #28 still a problem because this uses `@Value.Immutable`?

File: build.gradle (hunk 1 above)
Comment (mnazbro) [reply build.gradle]: I was actually having trouble with building this locally without adding bintray, I'll try to remove it and see if it isn't.

**Comment by mnazbro** [comment]
This should fix the problem. The answer given from the Immutables authors: https://github.com/immutables/immutables/issues/291#issuecomment-213574619

File: build.gradle (hunk 1 above)
Comment (mnazbro) [reply build.gradle]: I fixed this by changing the group in build.gradle instead.
